libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.tsx

```diff
@@ -59,11 +63,15 @@ export class ModelOverview extends React.Component<
   public context: React.ContextType<typeof ModelAssessmentContext> =
     defaultModelAssessmentContext;
   private featureComboBoxRef = React.createRef<IComboBox>();
+  private datasetCohortViewPivotKey = "datasetCohortView";
```

Comment:
this is for default value, correct?
Better to make it "static const", or put const outside of class to save few letters when referencing.
If changeable, it is better to use state.

Reply:
static const sounds good!

Reply:
Sorry, I meant outside the class. It doesn't change so const works fine.